zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/InterpreterFactoryTest.java

```diff
@@ -17,481 +17,50 @@
-    assertTrue(intpProperties instanceof Map);
-
-    // check if interpreter instance is saved as Properties in conf/interpreter.json file
-    Map<String, InterpreterProperty> properties = new HashMap<String, InterpreterProperty>();
-    properties.put("key1", new InterpreterProperty("key1", "value1", "type1"));
-    properties.put("key2", new InterpreterProperty("key2", "value2", "type2"));
-
-    interpreterSettingManager.createNewSetting("newMock", "mock1", new LinkedList<Dependency>(), new InterpreterOption(false), properties);
-
-    String confFilePath = conf.getInterpreterSettingPath();
-    byte[] encoded = Files.readAllBytes(Paths.get(confFilePath));
-    String json = new String(encoded, "UTF-8");
-
-    InterpreterInfoSaving infoSaving = InterpreterInfoSaving.fromJson(json);
-    Map<String, InterpreterSetting> interpreterSettings = infoSaving.interpreterSettings;
-    for (String key : interpreterSettings.keySet()) {
-      InterpreterSetting setting = interpreterSettings.get(key);
-      if (setting.getName().equals("newMock")) {
-        assertEquals(setting.getProperties().toString(), properties.toString());
-      }
-    }
-  }
-
-  @Test
-  public void testInterpreterAliases() throws IOException, RepositoryException {
-    interpreterSettingManager = new InterpreterSettingManager(conf, depResolver, new InterpreterOption(true));
-    factory = new InterpreterFactory(conf, null, null, null, depResolver, false, interpreterSettingManager);
-    final InterpreterInfo info1 = new InterpreterInfo("className1", "name1", true, null);
-    final InterpreterInfo info2 = new InterpreterInfo("className2", "name1", true, null);
-    interpreterSettingManager.add("group1", new ArrayList<InterpreterInfo>() {{
-      add(info1);
-    }}, new ArrayList<Dependency>(), new InterpreterOption(true), Collections.EMPTY_MAP, "/path1", null);
-    interpreterSettingManager.add("group2", new ArrayList<InterpreterInfo>(){{
-      add(info2);
-    }}, new ArrayList<Dependency>(), new InterpreterOption(true), Collections.EMPTY_MAP, "/path2", null);
-
-    final InterpreterSetting setting1 = interpreterSettingManager.createNewSetting("test-group1", "group1", new ArrayList<Dependency>(), new InterpreterOption(true), new HashMap<String, InterpreterProperty>());
-    final InterpreterSetting setting2 = interpreterSettingManager.createNewSetting("test-group2", "group1", new ArrayList<Dependency>(), new InterpreterOption(true), new HashMap<String, InterpreterProperty>());
-
-    interpreterSettingManager.setInterpreters("user", "note", new ArrayList<String>() {{
-      add(setting1.getId());
-      add(setting2.getId());
-    }});
-
-    assertEquals("className1", factory.getInterpreter("user1", "note", "test-group1").getClassName());
-    assertEquals("className1", factory.getInterpreter("user1", "note", "group1").getClassName());
-  }
-
-  @Test
-  public void testMultiUser() throws IOException, RepositoryException {
```

Comment:
Do we have an equivalent test for this?

Reply:
I think it is covered by https://github.com/apache/zeppelin/pull/2554/files#diff-63ebbe353cb86fc2eb25d29e14e7cb18R175